eu.usdx.UltraStarDeluxe.yaml

```diff
@@ -0,0 +1,194 @@
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DOPENCV_GENERATE_PKGCONFIG=ON
+    - -DBUILD_TESTS=OFF
+    - -DBUILD_opencv_apps=OFF
+    - -DBUILD_opencv_calib3d=OFF
+    - -DBUILD_opencv_dnn=OFF
+    - -DBUILD_opencv_features2d=OFF
+    - -DBUILD_opencv_flann=OFF
+    - -DBUILD_opencv_gapi=OFF
+    - -DBUILD_opencv_highgui=OFF
+    - -DBUILD_opencv_java_bindings_generator=OFF
+    - -DBUILD_opencv_ml=OFF
+    - -DBUILD_opencv_objdetect=OFF
+    - -DBUILD_opencv_photo=OFF
+    - -DBUILD_opencv_python2=OFF
+    - -DBUILD_opencv_python_bindings_generator=OFF
+    - -DBUILD_opencv_python_tests=OFF
+    - -DBUILD_opencv_stitching=OFF
+    - -DBUILD_opencv_ts=OFF
+    - -DBUILD_opencv_video=OFF
+    - -DCV_TRACE=OFF
+    - -DWITH_PROTOBUF=OFF
+    - -DWITH_FFMPEG=OFF
+    - -DWITH_ADE=OFF
+    - -DWITH_QUIRC=OFF
+    - -DWITH_GTK=OFF
+    - -DWITH_TIFF=OFF
+    - -DWITH_WEBP=OFF
+    - -DWITH_JASPER=OFF
+    - -DWITH_OPENJPEG=OFF
+    - -DWITH_PNG=OFF
+    - -DWITH_OPENEXR=OFF
+    - -DWITH_GDAL=OFF
+    - -DWITH_GDCM=OFF
+    - -DWITH_IMGCODEC_HDR=OFF
+    - -DWITH_IMGCODEC_SUNRASTER=OFF
+    - -DWITH_IMGCODEC_PXM=OFF
+    - -DWITH_IMGCODEC_PFM=OFF
+  cleanup:
+  - /share/opencv4/*cascades
+  - /share/opencv4/valgrind*.supp
+
+- name: projectm
+  sources:
+  - type: archive
+    url: https://github.com/projectM-visualizer/projectm/releases/download/v2.2.1/projectM-2.2.1.tar.gz
+    sha256: 9bbb33c5ba048537e97ea5ba2bd9fef76972c881597599a272b0194ec1d5f2a3
+  - type: patch
+    paths:
+    - patches/projectM-2.2.1.patch
+  buildsystem: autotools
+  rm-configure: true
+  config-opts:
+  - --disable-static
+  post-install:
+  - "f=$(fc-match -f '%{file}' sans) ; sed -i \"/Title Font/s:=.*:= $f:\" ${FLATPAK_DEST}/share/projectM/config.inp"
+  - "f=$(fc-match -f '%{file}' monospace) ; sed -i \"/Menu Font/s:=.*:= $f:\" ${FLATPAK_DEST}/share/projectM/config.inp"
+  cleanup:
+  - /share/projectM/fonts
+  - /bin
+
+- shared-modules/lua5.4/lua-5.4.json
+
+- name: portmidi
+  sources:
+  - type: archive
+    url: https://sourceforge.net/projects/portmedia/files/portmidi/217/portmidi-src-217.zip
+    sha256: 08e9a892bd80bdb1115213fb72dc29a7bf2ff108b378180586aa65f3cfd42e0f
+  - type: shell
+    commands:
+    - find -type f | while read a ; do tr -d '\r' < "$a" > /tmp/t && cat /tmp/t > "$a" ; done
+  - type: patch
+    paths:
+# Patches taken from Debian portmidi 217-6.1
+    - patches/00_cmake.diff
+    - patches/02_pmlinuxalsa.diff
+    - patches/03_pm_test_Makefile.diff
+    - patches/11-pmlinuxalsa.patch
+    - patches/13-disablejni.patch
+    - patches/20-movetest.diff
+    - patches/21-hardentests.diff
+    - patches/30-porttime_cmake.diff
+    - patches/40-test_sysex.diff
+    - patches/41-pm_linux.diff
+    - patches/50-change_assert.diff
+    - patches/51-remove_assert.diff
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  cleanup:
+  - "*.a"
+  - "libporttime.*"
+
+- name: ultrastardx
+  sources:
```

Comment:
sources is usually last in the module

Reply:
Not intuitive IMHO. Is this a hard requirement?

Reply:
Actually it's intuitive because that's where revieweres/maintainers expect it to be. It doesn't affect the build.